association.go

```diff
@@ -126,12 +126,13 @@ func getAssociationStateString(a uint32) string {
 //
 // Tag         :
 // State       : A state variable indicating what state the association
-//             : is in, i.e., COOKIE-WAIT, COOKIE-ECHOED, ESTABLISHED,
-//             : SHUTDOWN-PENDING, SHUTDOWN-SENT, SHUTDOWN-RECEIVED,
-//             : SHUTDOWN-ACK-SENT.
 //
-//               Note: No "CLOSED" state is illustrated since if a
-//               association is "CLOSED" its TCB SHOULD be removed.
+//	: is in, i.e., COOKIE-WAIT, COOKIE-ECHOED, ESTABLISHED,
+//	: SHUTDOWN-PENDING, SHUTDOWN-SENT, SHUTDOWN-RECEIVED,
+//	: SHUTDOWN-ACK-SENT.
```

Comment:
https://www.rfc-editor.org/rfc/rfc4960.html#section-13.2
If this indent is lost, it would be better to make it like
```go
// Peer Verification Tag:
//   Tag value to be sent in every packet and is received in the INIT or INIT ACK chunk.
//
// My Verification Tag: 
//   Tag expected in every inbound packet and sent in the INIT or INIT ACK chunk.
//
// State:
//   A state variable indicating what state the association
//   is in, i.e., COOKIE-WAIT, COOKIE-ECHOED, ESTABLISHED,
//   SHUTDOWN-PENDING, SHUTDOWN-SENT, SHUTDOWN-RECEIVED,
//   SHUTDOWN-ACK-SENT.
```

Reply:
@stv0g https://github.com/pion/rtp/pull/214